composer/json_schemas/generate_json_schemas.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Generate JSON Schemas for all Hparam classes."""
+
+import os
+
+# Dataset Hparams
+from composer.datasets.ade20k_hparams import ADE20kDatasetHparams, StreamingADE20kHparams
```

Comment:
Adding on to the comment below, I think we will only need to generate the JSONSchema for the experiment_hparams.py class, since all other hparam classes should be accessible within (that hparams class serves as the root, if you will).